CONTRIBUTING.md

```diff
@@ -97,7 +97,7 @@ For content changes, make sure that you:
 When you open a pull request, you must fill out the "Ready for review" template before we can review your PR. This template helps reviewers understand your changes and the purpose of your pull request.
 
 ### Python Styleguide
-All Python code is linted with [Black](https://github.com/psf/black) using the default settings. Your code will not be accepted if it is not blackened.
+All Python code is lined with [Black](https://github.com/psf/black) using the default settings. Your code will not be accepted if it is not blackened.
```

Comment:
Sorry, what It does mean: `lined`?
If we open https://github.com/psf/black , It's a code formatter.

Maybe It's more correct to write: **All Python code is formatted with**
What do you think?

Reply:
Yeah man, sorry about that. It was 3 am in my time, so I've probably misunderstood. Thank you for mentioning btw

Reply:
Thanks for your time and contribution :)